quickstarts/mern/config.yml

```diff
@@ -0,0 +1,63 @@
+
+# Name of the quickstart
+slug: mern-stack
+
+# Title of the quickstart
+title: MERN stack
+
+summary: |
+  This MERN quickstart allows you to monitor a MERN stack (MongoDB, ExpressJS, ReactJS, NodeJS).
+  MERN stack is a full stack software development kit. It is used to develop web applications. MERN is utilized by developers and businesses worldwide.
+  Our MERN integration includes a specific dashboard to get observability for an entire MERN stack.
+
+# Description of the quickstart
+description: |
+  ## Comprehensive MERN stack monitoring system:
+  MERN is a free and open-source JavaScript software stack for building dynamic web sites and web applications. Each of these 4 powerful technologies provides an end-to-end framework for the developers to work in and each of these technologies play a big part in the development of web applications.
+
+  ## What should you look for in a MERN dashboard?
+  Metrics of MongoDB, i.e. number of active connections, database details, usage time and many more.
+  Metrics of ExpressJS, NodeJS, i.e. number of web transactions, top transactions and the error transactions, memory usage and many more.
+  Metrics of ReactJS, i.e. number of viewed urls, page views by city, ajax requests and many more. You are also free to create your own chart by using the stored metrics.
+
+  ## What’s included in the MERN quickstart?
+  Install this quickstart to install preconfigured observability solutions:
+  - Multiple high-value alerts, including page views by city and ajax requests
+  - Informative dashboards (Top transactions, Error transactions, and more)
+
+# The logo of the quickstart
+icon: logo.png
+
+# Support level
+level: New Relic
+
+# Authors of the quickstart
+authors:
+  - New Relic
+  - Phaneendra Kosanam
+
+installPlans:
+  - mongodb-integration
+  - node-agent
+  - browser-docs
+
+keywords:
```

Comment:
If you want your quickstart to appear under a certain category in add data, please add specific keywords that will map to a specific category. The terms that appear match to the displayName.
[Query](https://api.newrelic.com/graphiql?#query=%7B%0A%20%20actor%20%7B%0A%20%20%20%20nr1Catalog%20%7B%0A%20%20%20%20%20%20categories%20%7B%0A%20%20%20%20%20%20%20%20terms%0A%20%20%20%20%20%20%20%20displayName%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A)